srcpkgs/eternity/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'eternity'
+pkgname=eternity
+version=4.02.00
+revision=1
+#archs="i686 x86_64"
+#wrksrc=
+create_wrksrc=yes
+build_wrksrc="${pkgname}-${version}"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="cmake"
+makedepends="SDL2-devel SDL2_mixer-devel SDL2_net-devel zlib-devel"
+short_desc="Advanced Boom-compatible Doom engine with vanilla compatibility"
+maintainer="Antkibo <hxd.1996@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/team-eternity/eternity"
+distfiles="https://github.com/team-eternity/eternity/archive/${version}.tar.gz https://github.com/Wohlstand/libADLMIDI/archive/v1.5.0.1-1.tar.gz"
+checksum="bb9f69af38394dadd5b4d619b4ce8930728e1e61c70085379f53978be228c333
+ f19b18f35564f4db6f5d35d820655ea48e2cf8ffb193037fad74eecbabb35f07"
+
+post_extract() {
+	cp -r ${wrksrc}/libADLMIDI-1.5.0.1-1/* ${wrksrc}/${pkgname}-${version}/adlmidi
```

Comment:
Should probably move the entire folder. `rmdir aldmidi` then `mv`.

Reply:
Thanks. I made the changes.